internal/proto/etcd_meta.proto

```diff
@@ -36,13 +36,20 @@ message CollectionInfo {
   int32 shards_num = 10;
   repeated common.KeyDataPair start_positions = 11;
   common.ConsistencyLevel consistency_level = 12;
-  repeated PartitionInfo partitions = 13;
+  repeated PartitionInfo partitions = 13; // deprecated.
```

Comment:
remove it and don't consider compatibility issues.

Reply:
removed.